.travis.yml

```diff
@@ -14,6 +14,8 @@ env:
   - VERSION=2.2 PYTHON_VERSION=3.6 PG_MAJOR=11 PG_VERSION=11.6-r0 VARIANT=alpine
   - VERSION=2.2 PYTHON_VERSION=3.7 PG_MAJOR=10 PG_VERSION=10.11-1.pgdg100+1 VARIANT=slim
   - VERSION=2.2 PYTHON_VERSION=3.7 PG_MAJOR=11 PG_VERSION=11.6-r0 VARIANT=alpine
+  - VERSION=2.2 PYTHON_VERSION=3.8 PG_MAJOR=10 PG_VERSION=10.11-1.pgdg100+1 VARIANT=slim
+  - VERSION=2.2 PYTHON_VERSION=3.8 PG_MAJOR=11 PG_VERSION=11.6-r0 VARIANT=alpine
```

Comment:
Not directly related to this line, but I think it may be worth using this as an opportunity to drop 2.0.x support from the matrix. Looks like extended support from Django ended in April 2019.

See: https://www.djangoproject.com/download/#supported-versions